public/components/overview/mitre_attack_intelligence/module_mitre_attack_intelligence_resource_flyout.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Wazuh app - React component for showing the Mitre Att&ck intelligence flyout.
+ *
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React , {useRef} from 'react';
+import { MitreAttackResources } from './mitre_attack_resources';
+import { GetFlyoutSubtable } from './module_mitre_attack_intelligence_resource_flyout_subTables';
+
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiOverlayMask,
+  EuiTitle,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlyoutBody,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { Markdown } from '../../common/util/markdown';
+
+export const ModuleMitreAttackIntelligenceFlyout = ({details, closeFlyout, tableProps}) => {
+  const startReference = useRef(null);
+  startReference.current?.scrollIntoView();
```

Comment:
I am not sure why the line `startReference.current?.scrollIntoView();`  is in each render